README.md

```diff
@@ -14,6 +14,75 @@ Thumbprint Tokens are published as JavaScript, SCSS, Swift, and Kotlin.
 
 You can use the [Thumbprint Tokens GraphQL server](https://thumbprint-tokens.netlify.com/) to query tokens and view the schema.
 
+Button, Theme, Title Color, Active, primary Tokens should follow this recommended hierarchy: `[<component-name>] <category-name> [<option-name> <state>] <token>`.
```

Comment:
I imagine we'll want to separate the component tokens from the non-component tokens in the doc. I wonder what is the best way to do that without having to hard-code the list of component tokens in the doc. 🤔